src/AutorestSwift/Models/Complex/Value.swift

```diff
@@ -128,4 +128,12 @@ class Value: Codable, LanguageShortcut {
         try container.encode(`protocol`, forKey: .protocol)
         if extensions != nil { try container.encode(extensions, forKey: .extensions) }
     }
+
+    var isSkipUrlEncoding: Bool {
+        var skipUrlEncoding: Bool?
+        if let value = extensions?["x-ms-skip-url-encoding"] {
+            skipUrlEncoding = value.value as? Bool
+        }
+        return skipUrlEncoding ?? false
```

Comment:
```suggestion
        return extensions?["x-ms-skip-url-encoding"]?.value as? Bool ?? false 
```

Reply:
Less concise, but still a simplification:
```suggestion
        var skipUrlEncoding = false
        if let value = extensions?["x-ms-skip-url-encoding"]?.value as? Bool {
            skipUrlEncoding = value
        }
        return skipUrlEncoding
```